include/roboteam_ai/bt/Node.hpp

```diff
@@ -12,10 +13,8 @@ namespace rtt {
     namespace ai {
     namespace world {
         class World;
-
-        class Field;
-
     }
+    class FieldComputations;
```

Comment:
fieldComputations is not namespaced?

Reply:
It is, but it is not namespaced in world. It is only namespaced in rtt::ai